math/exp.c

```diff
@@ -45,6 +49,11 @@ double ldexp(double x, int exp)
 	conv_t *conv = (conv_t *)&x;
 	int exponent = 0;
 
+	if (isnan(x) != 0) {
+		errno = EDOM;
```

Comment:
Does any standard/specification dictate that `errno` should be set to `EDOM` in this function?